src/js/tab-switcher.js

```diff
@@ -0,0 +1,27 @@
+import homePage from './home'
+import menuPage from './menu'
+import aboutPage from './about'
+
+function removeCurrentTab() {
```

Comment:
- [ ] [REQUIRED] Please  use arrow functions instead of the `function` keyword (this is only an example, all `function` keywords should be replaced)
![image](https://user-images.githubusercontent.com/73254040/99142509-1bf5d400-261b-11eb-95c4-53775e232778.png)
[Source](https://microverse.pathwright.com/auth/sign-in/?next=/library/fast-track-curriculum/69047/path/step/59622983/)

Reply:
Converted all traditional functions into arrow functions.